module-1/lab-errhand_listcomp/your-code/main.py

```diff
@@ -53,6 +53,11 @@
 
 #4. Create this list [-10,-9,-8,-7,-6,-5,-4,-3,-2,-1,0]. Use my_list as the name of the list.
 # Remember to use list comprehensions and to print your results
+
```

Comment:
Other two ways : 
my_list = [num for num in range(-10,1)]
results = [-num for num in reversed(range(11))]